packages/zoltan2/sphynx/src/Zoltan2_SphynxProblem.hpp

```diff
@@ -231,59 +232,87 @@ static void setSphynxValidatorsInList(
   ///////////////////////////////////////////////////////////////////////////
 
   template <typename Adapter>
-  void SphynxProblem<Adapter>::processAlgorithmName(
-      const std::string &algorithm, const std::string &defString,
-      const std::string &model, Environment &env, bool &removeSelfEdges,
-      bool &isGraphType, bool &needConsecutiveGlobalIds)
-  {
-    this->algName_ = std::string("sphynx");
+    void SphynxProblem<Adapter>::processAlgorithmName(
+        const std::string &algorithm, const std::string &defString,
+        const std::string &model, Environment &env, bool &removeSelfEdges,
+        bool &isGraphType, bool &needConsecutiveGlobalIds)
+    {
+      this->algName_ = std::string("sphynx");
+    }
+
+  /*template <typename Adapter>
+    void SphynxProblem<Adapter>::solve(Teuchos::RCP<mvector_t> &userEigenVects)
+    {
```

Comment:
We need to either make this work or remove it.

Reply:
I think it's best to remove it. I tried to debug the errors I got when I uncommented this code, but with no success